tools/create_db.py

```diff
@@ -448,7 +454,21 @@ def write_batch(self, batch):
         keys = self.get_keys(len(batch))
         lmdb_txn = self.db.begin(write=True)
         for i, datum in enumerate(batch):
-            lmdb_txn.put('%08d_%d' % (keys[i], datum.label), datum.SerializeToString())
+            try:
+                key = '%08d_%d' % (keys[i], datum.label)
+                lmdb_txn.put(key, datum.SerializeToString())
+            except lmdb.MapFullError:
+                lmdb_txn.abort()
+
+                # double the map_size
+                curr_limit = self.db.info()['map_size']
+                new_limit = curr_limit*2
+                logger.debug('Doubling LMDB map size to %sMB ...' % (new_limit>>20,))
+                self.db.set_mapsize(curr_limit*2) # double it
```

Comment:
Hi @gheinrich, I see the problem now. Somehow I committed the version bump to the release branch, not master. I've merged the single missing commit back to master. Thanks for catching!